src/api/status/public/index.html

```diff
@@ -54,15 +55,15 @@
           href=" https://demos.creative-tim.com/material-dashboard/pages/dashboard "
           target="_blank"
         >
-          <img src="assets/img/logo-ct.png" class="navbar-brand-img h-100" alt="main_logo" />
-          <span class="ms-1 font-weight-bold text-white">Material Dashboard 2</span>
+          <img src="./logo.svg"" class="navbar-brand-img h-100" alt="main_logo" />
+          <span class="ms-1 font-weight-bold text-white">Telescope Dashboard</span>
         </a>
       </div>
       <hr class="horizontal light mt-0 mb-2" />
       <div class="collapse navbar-collapse w-auto max-height-vh-100" id="sidenav-collapse-main">
         <ul class="navbar-nav">
           <li class="nav-item">
-            <a class="nav-link text-white active bg-gradient-primary" href="./pages/dashboard.html">
+            <a class="nav-link text-white active" href="./pages/dashboard.html">
```

Comment:
Clicking on Dashboard should take us to root: `./`

Reply:
Thank you 👍